### PR TITLE
Fix: ui_resource: change return code and error to warning for some unharmful actions

### PR DESCRIPTION
**Pull request by liangxin1300**

### CODE REVIEW

File: crmsh/ui_resource.py [hunk 1]
```diff
@@ -496,7 +496,8 @@ def ms_to_sec(m):
             return m[:len(m)-3]
 
         if rsc not in compl.resources():
-            context.fatal_error("Resource {} not exists in this cluster".format(rsc))
+            context.warning("Resource {} not exists in this cluster".format(rsc))
+            return
```
Comment (gao-yan) [inline crmsh/ui_resource.py:500]: This looks good to me.

File: crmsh/ui_resource.py [hunk 2]
```diff
@@ -515,7 +516,7 @@ def ms_to_sec(m):
             import re
             failcount_res = re.findall(r'fail-count-{}#(.*)_([0-9]+)'.format(rsc), out)
             if not failcount_res:
-                context.fatal_error("No failcount on node {} for resource {}".format(node, rsc))
+                return
```
Comment (gao-yan) [inline crmsh/ui_resource.py:519]: Strictly,  it probably should return success here only if the specified `value` is `0`.

**Review by gao-yan — COMMENTED** [review]
Thanks for the changes, @liangxin1300. Some comments in below.

File: crmsh/ui_resource.py (hunk 2 above)
Comment (liangxin1300) [reply crmsh/ui_resource.py:519]: I think it's fine to return success here, since no failcount here is unharmful

Comment (gao-yan) [reply crmsh/ui_resource.py:519]: Like said, it's not really an usual case that users manipulate failcounts and set them to non-zero value, but if crmsh happens to receive such a request and is refusing to set it to non-zero, it as least should tell users they are not getting the expected result .

Comment (liangxin1300) [reply crmsh/ui_resource.py]: Thanks for pointing this @gao-yan ! Already changed as `return True if value and int(value) == 0 else False`

**Review by gao-yan — APPROVED** [review]

**Pull request merged**